examples/demo-app/core/src/multiSubmit.ts

```diff
@@ -113,6 +113,7 @@ export const multiSubmit = command({
     const maxConcurrency = 5;
```

Comment:
5 still seems like a lot of nonces to be in flight at once.  I'd be a bit nervous with more than 2.  Or maybe we can make this an option?

Reply:
Yeah, I'll make this an option with default value 2